pages/common/wfuzz.md

```diff
@@ -3,18 +3,22 @@
 > A web application bruteforcer.
 > More information: <https://wfuzz.readthedocs.io/en/latest/user/basicusage.html>.
 
-- Directory and file bruteforce using the specified wordlist and also proxying the traffic:
+- Directory and file bruteforce using the specified [w]ordlist and also [p]roxying the traffic:
 
 `wfuzz -w {{path/to/file}} -p {{127.0.0.1:8080}} {{http://example.com/FUZZ}}`
 
-- Save the results to a file:
+- Save the results to a [f]ile:
 
 `wfuzz -w {{path/to/file}} -f {{filename}} {{http://example.com/FUZZ}}`
 
-- Show colorized output while only showing the declared response codes in the output:
+- Show [c]olorized output while only showing the declared response codes in the output:
 
 `wfuzz -c -w {{path/to/file}} --sc {{200,301,302}} {{http://example.com/FUZZ}}`
 
-- Use a custom header to fuzz subdomains while hiding specific response codes and word counts. Increase the threads to 100 and include the target ip/domain:
+- Use a custom [H]eader to fuzz subdomains while hiding specific response codes and word counts. Increase the [t]hreads to 100 and include the target ip/domain:
 
 `wfuzz -w {{path/to/file}} -H {{"Host: FUZZ.example.com"}} --hc {{301}} --hw {{222}} -t {{100}} {{example.com}}`
+
+- Brute force Basic Authentication using a list of usernames and passwords from files, [h]iding response [c]odes of unsuccessful attempts:
```

Comment:
```suggestion
- Brute force Basic Authentication using a list of usernames and passwords from files for each FUZ[z] keyword, [h]iding response [c]odes of unsuccessful attempts:
```

Reply:
FUZ[z] does not fit into the style for multiple placeholders for `wfuzz`.

1st placeholder: FUZZ
2nd placeholder: FUZ2Z
3rd placeholder: FUZ3Z

So after the first FUZZ, the subsequent placeholders are FUZnZ where n >= 2

Here is a ref: https://www.hackingarticles.in/a-detailed-guide-on-wfuzz/
Search for **Double fuzzing** on the weblink above

Reply:
It does fit on the style, because it refers to the z mnemonic refers to `-z`. However, I agree it can be improved.